### PR TITLE
Add pipe bundles

### LINKED ISSUE #56830: Add bundles of pipe
### Is your feature request related to a problem? Please describe.

Similarly to #46551, shifting pipes one by one is annoying. They are easily bundlable, similar to planks, by use of wire or rope.

### Solution you would like.

A recipe for bundles of pipe, similar to bundling planks.

### Describe alternatives you have considered.

Keep plodding around with piles of plumbing paraphernalia like a perverse pariah (betcha can't say that three times fast).

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Content "Add a bundle of pipes"

#### Purpose of change
Unlike all other large, bulky materials, pipes cannot be tied into bundles for ease of transportation. This pull request simply adds a bundle of ten pipes, along with ways to assemble and disassemble it.

#### Describe the solution
Using ten pipes and some variety of rope, the player can tie the pipes into a bundle. When finished, it can be deconstructed back into its base components.

#### Describe alternatives you've considered
None

#### Testing
Made a bundle of pipes, carried it around, then disassembled it. No bugs or errors resulted.

#### Additional context
Fixes #56830
Obsoletes #56997

### CODE REVIEW

**Comment by Termineitor244** [comment]
If merged, this PR would obsolete #56997 and fix #56830.

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -4907,6 +4907,15 @@
     "components": [ [ [ "felt_patch", 10 ] ] ],
     "flags": [ "BLIND_EASY" ]
   },
+  {
+    "result": "bundle_pipe",
+    "type": "uncraft",
```
Comment (Maleclypse) [inline data/json/recipes/recipe_deconstruction.json:4912]: I don't think you need the uncraft if you made the recipe reversible and this uncraft will destroy whatever rope you used to craft it while reversible recipes should uncraft back into their used components whatever they were.

Comment (MNG-cataclysm) [reply data/json/recipes/recipe_deconstruction.json]: I have deleted the uncraft recipe.

**Comment by TheShadowFerret** [comment]
A copper tubing bundle would also be appreciated ❤️

**Comment by MNG-cataclysm** [comment]
> A copper tubing bundle would also be appreciated ❤️

Copper tubing bundles have been added.

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**